content/sensu-go/5.16/api/assets.md

```diff
@@ -28,22 +28,33 @@ The `/assets` API endpoint provides HTTP GET access to [asset][1] data.
 The following example demonstrates a request to the `/assets` API endpoint, resulting in a JSON array that contains [asset definitions][1].
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/assets -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET \
+http://127.0.0.1:8080/api/core/v2/namespaces/default/assets \
+-H "Authorization: Bearer $SENSU_TOKEN" \
 
 HTTP/1.1 200 OK
 [
+  {
+    "url": "https://github.com/sensu/sensu-influxdb-handler/releases/download/3.1.2/sensu-influxdb-handler_3.1.2_linux_amd64.tar.gz",
+    "sha512": "612c6ff9928841090c4d23bf20aaf7558e4eed8977a848cf9e2899bb13a13e7540bac2b63e324f39d9b1257bb479676bc155b24e21bf93c722b812b0f15cb3bd",
+    "filters": null,
+    "builds": null,
```

Comment:
do we need to include filters and builds at all if they're going to be `null`?  might be better to just remove them instead?

Reply:
Another question is whether you can create assets with filters (i.e., can you add filters later if you exclude them when you initially create an asset?).

I could not get the calls to succeed with a filter specified. The calls to create an asset do work with filters:null or leaving the filters array out altogether.

Reply:
Cameron's filters format suggestion worked, so this is fixed now!